python/src/scipp/coords.py

```diff
@@ -68,6 +72,24 @@ def _produce_coord(obj, name):
     return obj.coords[name]
 
 
+def _move_to_back(lis, val):
+    lis.remove(val)
+    lis.append(val)
+    return lis
+
+
+def _store_coord(obj, name, coord):
+    edges = find_bin_edges(coord, obj)
+    if not edges:
+        obj.coords[name] = coord
+    elif len(edges) == 1:
+        obj.coords[name] = coord.transpose(_move_to_back(coord.dims, edges[0]))
```

Comment:
General purpose here is to make the bin edge coord tin inner most. Maybe comment to that effect or create a corresponding named function to make it clearer what's going on (and why)

Reply:
Good idea, but this change is actually part of #2086 I based #2087 on that PR.